ARK Server Manager/ServerSettingsControl.xaml.cs

```diff
@@ -548,6 +554,100 @@ private void MaxXPDino_Reset(object sender, RoutedEventArgs e)
             this.Settings.ResetOverrideMaxExperiencePointsDino();
         }
 
+        public ICommand ExportActionCommand
+        {
+            get
+            {
+                return new RelayCommand<ServerSettingsImportExportAction>(
+                    execute: (action) =>
+                    {
+                        try
+                        {
+                            var dialog = new CommonSaveFileDialog();
+
+                            switch (action)
+                            {
+                                case ServerSettingsImportExportAction.DinoLevels:
+                                    dialog.Title = "Save level customizations";
```

Comment:
All of these dialogs are very similar except for some text it looks like.  Consider refactoring to a method which generates the dialog from the parameters.